mesos/Makefile

```diff
@@ -40,11 +40,17 @@ mesos: check-version
 	  sed -i -e "s/{UBUNTU_TESTING}//g" $@ ; \
 	fi
 	docker build -t mesosphere/$@:$(MESOS_TAG) - < $@
+	$(eval IMAGE_ID=$(shell docker images | grep 'mesosphere/$@ ' | grep $(MESOS_TAG) | awk '{print $$3}'))
+	docker tag $(IMAGE_ID) mesosphere/$@:latest
```

Comment:
Given that we are never pushing the image with `latest` tag any specific reason we are actually creating an image with that tag?